py/private/py_image_layer.bzl

```diff
@@ -141,24 +130,11 @@ def py_image_layer(
 
     # Produce the manifest for a tar file of our py_binary, but don't tar it up yet, so we can split
     # into fine-grained layers for better pull, push and remote cache performance.
-    manifest_name = name + ".manifest"
```

Comment:
hmm why change this? we recently landed a change to fix this; https://github.com/bazel-contrib/bazel-lib/commit/8a9204ab17cbe337a64e838dd193b00f243764fb i think we should keep this as is.